src/styles/item-image-styles.ts

```diff
@@ -7,6 +7,7 @@ export const baseItemImageStyles = css`
   .drop-shadow {
     filter: drop-shadow(1px 1px 2px rgba(0, 0, 0, 0.8));
     overflow: hidden;
+    height: 160px;
```

Comment:
blocker: There should not be fixed values in this component. 

Please change to `100%`

Reply:
fixed